flocx_ui/test/integration/pages/admin/other/flocxpage.py

```diff
@@ -0,0 +1,7 @@
+from openstack_dashboard.test.integration_tests.pages import basepage
+
+
+class FlocxPage(basepage.BaseNavigationPage):
+    def __init__(self, driver, conf):
+        super(FlocxPage, self).__init__(driver, conf)
+        self._page_title = "Flocx"
```

Comment:
Add a new line. It is a nice practice to add '\n' which will help with diffs and EOF later.